go/vt/vtctl/vtctl.go

```diff
@@ -2257,6 +2257,7 @@ func commandValidateSchemaShard(ctx context.Context, wr *wrangler.Wrangler, subF
 func commandValidateSchemaKeyspace(ctx context.Context, wr *wrangler.Wrangler, subFlags *flag.FlagSet, args []string) error {
 	excludeTables := subFlags.String("exclude_tables", "", "Specifies a comma-separated list of tables to exclude. Each is either an exact match, or a regular expression of the form /regexp/")
 	includeViews := subFlags.Bool("include-views", false, "Includes views in the validation")
+	allowEmpty := subFlags.Bool("allow-empty", false, "Allow empty shards when validating")
```

Comment:
What does an empty shard mean? Is it one with no master, or no tablets at all?

Reply:
I'm thinking a tablet with no tablets at all, but I'm assuming that if there is no master, there a no tablets. 

Should we name this differently ?

Reply:
If we mean no tablets, the name is fine, but if we mean no master, we should probably rename to `allowNoMaster`.

Reply:
the flag description should also be updated to make the meaning clear.